encord/objects/classification_instance.py

```diff
@@ -482,15 +497,20 @@ class FrameData:
 
         @staticmethod
         def from_dict(d: dict) -> ClassificationInstance.FrameData:
-            if "lastEditedAt" in d:
+            if "lastEditedAt" in d and d["lastEditedAt"] is not None:
                 last_edited_at = parse_datetime(d["lastEditedAt"])
             else:
                 last_edited_at = datetime.now()
 
+            if "createdAt" in d and d['createdAt'] is not None:
+                created_at = parse_datetime(d["createdAt"])
+            else:
+                created_at = datetime.now()
+
             return ClassificationInstance.FrameData(
-                created_at=parse_datetime(d["createdAt"]),
+                created_at=created_at,
                 created_by=d["createdBy"],
-                confidence=d["confidence"],
+                confidence=d.get("confidence", DEFAULT_CONFIDENCE), # TODO QUESTION: Is setting a default alright here?
```

Comment:
i think it's alright as safety net, as we should expect it to be filled anyways, iirc